content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -653,7 +646,7 @@ pod-to-node assignment to kube-scheduler. This is allowed only for suspended Job
 been unsuspended before.
 
 The fields in a Job's pod template that can be updated are node affinity, node selector, 
-tolerations, labels and annotations.
+tolerations, labels, annotations and scheduling gates.
```

Comment:
nit: can we make scheduling gates into a hyperlink?

Reply:
done.